cogs/games.py

```diff
@@ -23,15 +23,230 @@
 
 # Other utilities
 import re
+import os
+from time import time
+import pickle
+import random
+from asyncio import TimeoutError
+from typing import Dict, List, Set, Tuple
 from .utils.dice_roll import dice_roll
 from .utils.clamp_default import clamp_default
+from .utils.hangman import LOSS_MISTAKES, mk_hangman_str
+from .currency import HANGMAN_REWARD
 
 ROLL_PATTERN = re.compile(r'^(\d*)d(\d*)([+-]?\d*)$')
+HM_WIN_CHEEPS = 10
+HM_COOL_WIN_CHEEPS = 20
```

Comment:
these 2 should ideally be put in config.ini along with other currency-tweaking params